projects/canopy/src/lib/promo-card/promo-card.notes.ts

```diff
@@ -0,0 +1,135 @@
+})
+~~~
+
+and in your HTML, place your Promo Cards in your Promo Card List:
+
+~~~html
+<lg-promo-card-list>
+  <lg-separator [variant]="dotted" lgMargin="none"></lg-separator>
+  <lg-promo-card-list-title [headingLevel]="headingLevel">
+    Get more from Legal &amp; General
+  </lg-promo-card-list-title>
+  <lg-promo-card
+    [variant]="variant">
+    <lg-promo-card-image [imageUrl]="imageUrl"></lg-promo-card-image>
+    <lg-promo-card-title [headingLevel]="headingLevelInner">
+      Need help with Care?
+    </lg-promo-card-title>
+    <lg-promo-card-content>
+      <p>Our care service can help you understand, find and fund care for you or a loved one.</p>
+    </lg-promo-card-content>
+    <lg-promo-card-footer>
+      <button
+        lgMarginBottom="none"
+        lg-button
+        type="button"
+        [variant]="variant">
+        Find out more
+      </button>
+    </lg-promo-card-footer>
+  </lg-promo-card>
+</lg-promo-card-list>
+~~~
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| \`\`headingLevel\`\` | Allows the card (list) heading level to be set | number | n/a | Yes |
+| \`\`variant\`\` | The variant of promo card: \`\`solid-white\`\`, \`\`solid-green\`\` | PromoCardVariant | 'solid-white' | No |
+| \`\`imageUrl\`\` | Identifies the card image | string | n/a | Yes |
+
+## Using only the SCSS files
+
+Generate the markup as shown in the example below. Please ensure you have imported the necessary SCSS files.
+
+This example contains only one promo card. Repeat as necessary.
+
+| Class | Description |
+|------|-------------|
+| \`\`lg-promo-card-list\`\` | Adds styles to the outer promo card list element |
+| \`\`lg-promo-card-list__separator\`\` | Adds styles to the promo card list separator |
+| \`\`lg-promo-card-list-title\`\` | Adds styles to the promo card list title |
+| \`\`lg-promo-card-list-title__heading\`\` | Adds styles to the promo card list title inner heading |
+| \`\`lg-promo-card-list__promo-cards\`\` | Adds styles to the promo card list cards wrapper |
+| \`\`lg-promo-card\`\` | Adds styles to the promo card |
+| \`\`lg-promo-card__img-wrapper\`\` | Adds styles to the promo card image wrapper |
+| \`\`lg-promo-card-image\`\` | Adds styles to the promo card image |
+| \`\`lg-promo-card-image__content\`\` | Adds styles to the promo card image content |
+| \`\`lg-promo-card__content-wrapper\`\` | Adds styles to the promo card content wrapper |
+| \`\`lg-promo-card__content-inner\`\` | Adds styles to the promo card inner content |
+| \`\`lg-promo-card-title\`\` | Adds styles to the promo card title |
+| \`\`lg-promo-card-title__heading\`\` | Adds styles to the promo card title inner heading |
+| \`\`lg-promo-card-content\`\` | Adds styles to the promo card content |
+| \`\`lg-promo-card-footer\`\` | Adds styles to the promo card footer |
+| \`\`lg-btn__text\`\` | Adds styles to the promo card footer CTA button |
```

Comment:
I think you can leave this particular line out as it's tracked in the button docs already. The more I see us writing the 'Using only the SCSS' docs, the more I think we need to automate it away somehow.